packages/bentley/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@bentley/grouping-mapping-widget",
+  "version": "0.0.15",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/main.bundle.js",
+  "types": "lib/grouping-mapping-widget.d.ts",
+  "scripts": {
+    "build": "webpack && npm run copy-files ",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "copy-files": "ncp README.md lib/README.md && ncp LICENSE.md lib/LICENSE.md",
+    "cover": "",
+    "lint": "npm run lint:ts",
```

Comment:
why is a separate command needed for this?

Reply:
we had a more involved linting setup in our previous repo, just an artifact of that, will remove